Sources/StreamChatUI/ChatMessageList/RETHINK/MessageCell.swift

```diff
@@ -29,10 +29,12 @@ class MessageCell<ExtraData: ExtraDataTypes>: _CollectionViewCell, UIConfigProvi
     var reactionsView: _ReactionsCompactView<ExtraData>?
     var errorIndicatorView: _ChatMessageErrorIndicator<ExtraData>?
 
-    var threadReplyCountLabel: UILabel?
+    var threadReplyCountButton: UIButton?
     var threadAvatarView: ChatAvatarView?
     var threadArrowView: _SimpleChatMessageThreadArrowView<ExtraData>?
     
+    var onThreadReplyTapped: (() -> Void)!
+    
```

Comment:
I think @evsaev already solved this? 🤔

Reply:
Yeah, this action has already been forwarded to the controller via **MessageCell.Delegate.didTapOnThread**. Now it's just the matter of changing the implementation of `handleTapOnThread(forCellAt:)` in `MessageListVC`

Reply:
Let's merge the correct long press implementation first, then I will rebase it and connect the implementation with the above solution